test/Ocelot.AcceptanceTests/RoutingTests.cs

```diff
@@ -445,18 +446,22 @@ public void should_return_response_200_when_host_has_trailing_slash()
                 .BDDfy();
         }
 
-        [Fact]
-        public void should_return_ok_when_upstream_url_ends_with_forward_slash_but_template_does_not()
+        [Theory]
+        [InlineData("/products")]
+        [InlineData("/products/")]
+        [Trait("Issue","Should fix issue 649")]
```

Comment:
@raman-m Here I am using Trait attribute here with `Issue` key.
Trait have key,value feature which provides easy way to filter out issue category verification test cases.
I am not writing another test case with method name "should_fix_issue_649", cause it will be eventually same by execution.
We can categorize with `Issue` key when we do verification from checkbox filter in test explorer.
![image](https://github.com/ThreeMammals/Ocelot/assets/29522704/a98316a9-154c-440a-a9c4-5e3d868ac624)

Reply:
What are your thoughts on it @raman-m ?

Reply:
Nice feature of xUnit! Make sense to use...

Reply:
Officially it is better to have separate tests for the issue to close it finally